app/include/lwip/app/dhcpserver.h

```diff
@@ -23,9 +23,7 @@ typedef struct dhcps_msg {
         uint8_t chaddr[16];
         uint8_t sname[64];
         uint8_t file[128];
-	// Recommendation from Espressif:
-	// To avoid crash in DHCP big packages modify option length from 312 to MTU - IPHEAD(20) - UDPHEAD(8) - DHCPHEAD(236).
-        uint8_t options[IP_FRAG_MAX_MTU - 20 - 8 - 236];
+        uint8_t options[312];

```

Comment:
Why don't use `DHCP_MSGOPTIONS_MIN_LEN` here (or remove that one)? Ain't that the same?